docker-compose.yml

```diff
@@ -220,10 +220,12 @@ services:
     networks:
       - yas-network
   postgres:
-    image: debezium/postgres:15-alpine
+    image: debezium/postgres

```

Comment:
I think we should fix the version of dockers images to prevent auto getting new version which can be incompatible. Any upgrade should be explicitly checked.

Reply:
I already custom again the docker image of debezium to add plugin wal2json, so the version of image in docker-compose file is minor you can see the build option, because the version cause confuse with developer that is image will be download from dockerhub instead build from local so I not added the version
```
build: ./docker/postgres
```

Reply:
Then maybe we should tag it with different name

Reply:
Thank for your suggestion I have added the version for postgres debezium custom 
```
image: debezium/postgres:15-alpine-custom
```